checkpoint/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM continuumio/miniconda3
```

Comment:
We should avoid having a distinct Docker image for comparing checkpoints if possible. A bunch of reasons for this, but the #1 reason is that we want the accuracy metrics to test something as close as possible to what actually runs in production envs. This means we should test against the real `mentorpal-classifier` image unless that's really impractical for some reason[s]